frontend/src/store/sliceSettingsStore.js

```diff
@@ -5,14 +5,13 @@ const state = {
   ScalebottomSheetOpen: false,
   TranslatebottomSheetOpen: false,
   PrintProfilebottomSheetOpen: false,
-  MachineSelectionOpen: false,
```

Comment:
You reverted a lot of changes I made. Was that on purpose?

Reply:
the only change I intended to do was changing the mutation SET_PROFILE_PRESET

Reply:
export const fuzzySkinList = ['None', 'Contour', 'Contour and hole', 'All walls'] I saw these list in the code and did it like that.  also the image that Neil said it Liked that. I am not telling I am wrong, I am just telling why I did like that.
![image-2](https://github.com/user-attachments/assets/3ff332d6-c12f-4a6f-997f-6d3f6c7ba1ba)